backend/src/main/java/com/summerHack/diningTogether/utils/EmailVerificationUtilsImpl.java

```diff
@@ -0,0 +1,55 @@
+package com.summerHack.diningTogether.utils;
+
+import com.summerHack.diningTogether.model.User;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+@Component
+public class EmailVerificationUtilsImpl implements EmailUtils{
```

Comment:
It looks like a service instead of "utils". Because usually utils only have static methods and do not use dependency injection.